### ISSUE TITLE
On the importance of Persistence in the Documents

### ISSUE THREAD

**Issue by rkoshak** [openhab/openhab-distro#262]
Continuing the discussion from [here](https://github.com/openhab/openhab-distro/pull/255#issuecomment-237503433) about whether Persistence should be listed at the end to bring Rules up in the list.

I think the summary arguments are: 

* users want to get to Rules as soon as possible so that section should be closer to the to top
* persistence is often overlooked and ignored but a key and important capability, and it is one piece of foundational information to fully understand rules

> I think persistence is a function not all users will use. In your scenario: "Once you have Items you can create a Sitemap. Oops, what happens when I restart? Don't care because the device is on 24/7, that's a problem that I can think of later. More importantly, let's finally have a smart smarthouse! rules! rules! rules! ... Almost finished? Well now that I think of it, it would be nice to have a chart with the temperatures in my sitemap." That's what I think can be compared with most other documentations where things like Backup&Restore or Troubleshooting are at the end of the documentation, even though a user might want to look into them sooner.

My position is that persistence is a function that all users **should** use, at least until such time that they have enough knowledge and experience to actually make the decision to use it or not. I've helped far too many people on the forum who are frustrated primarily from an ignorance about what persistence does. In my perfect world, OH would come with a MapDB restoreOnStartup for all Items persistence configuration on by default. Short of that, placing Persistence above Rules in the table of contents is a way to stress its importance.

The entire documents are hypertext, not a physical book, so users who jump to the conclusion that they don't need persistence will still jump ahead. 

>Maybe the reason a lot of users (can't speak from personal experience) didn't know about the possibilities of persistence should be tackled by giving the menu entry a better (less technical) name and by having a nice short configuration base article explaining in a few bullet points what is possible connected with links to the in-detail articles.

I agree, many of the section titles need to be reworked and Persistence is not necessarily as meaningful as Rules may be. However, "Persistence" is the technical term for the capability in OH and I don't want to have to talk around it. Perhaps subtitles to the sections would help:

Things and Items: Representing the Real World in openHAB
Sitemaps: The User Interface to your Home Automation
Persistence: Saving and Accessing Historic Data
Rules and Scripts: Defining Home Automation Behaviors

>having a nice short configuration base article explaining in a few bullet points what is possible connected with links to the in-detail articles.

This sounds like a good approach across the board.

**Comment by kaikreuzer** [comment]
I would agree and am also considering to make https://github.com/eclipse/smarthome/pull/1956 a part of the standard openHAB package, once it is merged and available.
And maybe we can also come up with some setting that by default persists stuff and users rather have to deactivate/re-configure it, if this is not what they want.

But my main question is: Why do you post this here as an issue? What do I have to do to be allowed to close it again? :-)
I think this is rather some discussion that should be in the forum instead...

**Comment by rkoshak** [comment]
It was a misreading of the following line, I thought I was being asked to open an issue to discuss this. Clearly I am mistaken and now I see the new issue was in reference to the rename of the "Working with openHAB section"

>I you want to further discuss this topic and also to answer kai's question for the section name: Would you like to open an issue for that?

Closing the issue, will move to the forum.
